lib/api/src/sys/import_object.rs

```diff
@@ -16,6 +17,12 @@ pub trait LikeNamespace {
     fn get_namespace_export(&self, name: &str) -> Option<Export>;
     /// Gets all exports in the namespace.
     fn get_namespace_exports(&self) -> Vec<(String, Export)>;
+    /// Returns the contents of this namespace as an `Exports`.
+    ///
+    /// This is used by `ImportObject::get_namespace_exports`.
+    fn as_exports(&self) -> Option<Exports> {
+        None
+    }
```

Comment:
Why `Option`? Also, why a default impl?

Reply:
For backward compatibility with any existing implementations of `LikeNamespace` outside wasmer.

Reply:
If backwards compatibility was not a concern then I would remove `LikeNamespace` entirely and use `Exports` directly in `ImportObject`.

Reply:
Makes sense